assessment.rb

```diff
@@ -1,12 +1,59 @@
 require 'csv'
 # Assessment answers to:
 
+#question 1:
+$ rails g model Licensee surname:string given_name:string gender:string height:integer weight:integer dob:date eye_color:string hair_color:string
+    invoke active_record
+    create db/migrate/<time stamp>_create_licensees.rb
+    create app/models/licensee.rb
+
```

Comment:
Yep, that looks right. I'm assuming you actually ran it in order to test it, right?